src/components/modal/modal.js

```diff
@@ -112,6 +113,12 @@ class Modal extends React.Component {
      * @type {Boolean}
      */
     this.listening = false;
+    this.state = { state: 'closed' };
```

Comment:
iit could be open by default. I'd also add some comments to this to explain what it is used for

Reply:
open is a defaultProp and is set to false.
What would you suggest to be the best wait to mirror the component that loads this?